### PR TITLE
The second large cleanup of Base

### PR DESCRIPTION
**Pull request by Yuki2718**
```
! ---------- Redundant rules
novelasesp.blogspot.de,novelasesp.blogspot.com,novelasesp.blogspot.com.au,novelasesp.blogspot.pe###HTML2

letfap.com##iframe[src^="https://syndication.exoclick.com/"]

! history.com##.m-in-content-ad-row
history.com##aside.m-in-content-ad-row

! hdzog.com##.player-showtime-two
hdzog.com##.player-wrapper > .player-showtime-two.without-title

! 2conv.com,flvto.biz,flv2mp3.by##.square-ads
! 2conv.com,flvto.biz,flv2mp3.by##.horizontal-ads
flvto.biz##.square-ads
flvto.biz##.horizontal-ads

! healthline.com##.css-1031od0
healthline.com##header#site-header + section.css-1031od0

! feetporno.com,freehardcore.com,deviants.com,webvoyeur.com##.twocolumns > div.aside
deviants.com##.twocolumns > div.aside > script[src*="/api/spots/"] + div[align="center"]
deviants.com##.twocolumns > div.aside > div[style*="text-align: left;"]
deviants.com##.twocolumns > div.aside > div[style*="text-align: left;"] ~ font

! download.pixelexperience.org,hackintoshzone.com##.adsbygoogle
!+ PLATFORM(ext_ff, ext_opera, ios, ext_android_cb, ext_ublock)
download.pixelexperience.org##.adsbygoogle

! tube8.net,tube8.es,tube8.com###result_container_fake
tube8.es###result_container_fake

! fetishshrine.com##.twocolumn > .r-col
fetishshrine.com##.video-view.recently-movies > .twocolumn > .r-col

! movierulzfree.me,bmoviesfree.ru,ffmovies.ru,fmovies.cool,putlockertv.to,putlockers9.ws,bmovies.se,fmovies.unblocked.vc,fmovies.se,fmovies.is,fmovies.se##.hd-buttons
fmovies.is##div[class="widget"] > div[class="row hd-buttons"]

! imore.com##.adunit
forums.imore.com##.adunit

! majesy.org,majesy.com,magesy.eu,magesy.blog,magesy.pw,informer.com,addictinginfo.com,cmacapps.com,computerera.co.in,cypt.ga,dailyuploads.net,download.mokeedev.com,fas.li,file-upload.cc,genbird.com,getlinkgenerator.com,realses.com,shink.in,upload.af,uploads.to,uplod.ws,urlex.org##.adsbygoogle
!+ PLATFORM(ext_ff, ext_opera, ios, ext_android_cb, ext_ublock)
majesy.com##.adsbygoogle

! ||xozilla.com/player/html.php
||xozilla.com/player/html.php?aid=pause_html&video_id=*&*referer=

! ||mylust.com/*.jsx
||mylust.com/4wkjbnwgz6qz.jsx

! ||exe.io/files/p*.js
||exe.io/files/p10.js

! ||zbporn.*/zpi/zp.js
||zbporn.com/zpi/zp.js

! ||js.hdzog.com/saber/
||js.hdzog.com/saber/hdzog/dutti/

! ||kickassanime.io/codea/
||kickassanime.io/codea/720x90*.html

! ||sexu.com/ifrm/
||sexu.com/ifrm/*?ad=awn


! ---------- Dead domains
||arcaukalb.site^
||ashbowany.site^
||aidalbark.site^
||1xredjzldy.world^
||snyowx29.com^$third-party
||khlgauqt2000.com^$third-party
||adtify.pl^$third-party
||myjj.pyzkk.com^$third-party
||201689.qingkan520.com^
||api.digitalbees.it^$third-party
||adscpm.site^$third-party
||zxwiejfbyr.com^
||drogomet.com^
||click-cpa.net^$third-party
||adshost.site^$third-party
||doopads.com^$third-party
||2778255fe56.com^
||hanaprop.com^$third-party
||mrperfect.in^$third-party

||video.cars247.tv^
||delivery.content-recommendation.net^$third-party
||apps2.ctnetpass2.com^$third-party
||appsha2.ctnetload2.com^$third-party
||gpbqs.24590.space^$third-party
||olswr.27684.club^$third-party
||d269y12mnftu9c.cloudfront.net^
||d2hluuq2g8gwrv.cloudfront.net^
||dpz55hrzd89zs.cloudfront.net^
||d2pc0q32fod0ha.cloudfront.net^
||ads.contractoruk.com^
||d3l4qa0kmel7is.cloudfront.net^
||d5ou3dytze6uf.cloudfront.net^
||d1vh0xkmncek4z.cloudfront.net^
||d26r15y2ken1t9.cloudfront.net^
||d11eq81k50lwgi.cloudfront.net^
||ddyv8sl7ewq1w.cloudfront.net^
||d3i1asoswufp5k.cloudfront.net^
||dc44qjwal3p07.cloudfront.net^
||dv2m1uumnsgtu.cloudfront.net^
||d1mxvenloqrqmu.cloudfront.net^
||dfrs12kz9qye2.cloudfront.net^
||dgkytklfjrqkb.cloudfront.net^
||jizvehd.makeuseof.com^
||d17rlarvg2khuc.cloudfront.net^
||d1ewpr7kbabyrj.cloudfront.net^
||d1gp9nlx229wzz.cloudfront.net^
||d1k29rhvz38kg5.cloudfront.net^
||d1v6js7bjzmhoa.cloudfront.net^
||d1vwpe7grtcv9d.cloudfront.net^
||d258hxk86e2ixw.cloudfront.net^
||d2i54aseqwhx68.cloudfront.net^
||d2l8bbn629wykr.cloudfront.net^
||d2nn3xyicdpsrf.cloudfront.net^
||d2ue9k1rhsumed.cloudfront.net^
||d3laygk9zni6hc.cloudfront.net^
||d3m79ugzs2d8im.cloudfront.net^
||da5w2k479hyx2.cloudfront.net^
||dm8ojo46iwxm2.cloudfront.net^
||dohs95d6tfj19.cloudfront.net^
||dqqulesm3pfse.cloudfront.net^
||dviixeyykyqjv.cloudfront.net^
||d2t2on23g4lty1.cloudfront.net^$third-party

animeheaven.eu
animeupdate6.pro
bitcomania.io
counsellingresult2016.in
dasffad2590.ramrem.space
forbes.uol.com.br
funnyquiz.blog
linksgenerator.ga
megapelishdonline.website
ngantukberat.me
repo.xarold.com
safebit.co.in
semimkv.net
techwidetoday.site
darkcomet.net
firmgoogle.com
apprendre-les-runes.com
arenavision2017.cf
dicaspravida.life
file4go.me
hibrido1337.blogspot.com
indextred.xyz
newsdust9.live
oldjogos.com
rl-hackers.net
files.love
2url.io
artiicle.xyz
bollyleague.com
craftz.xyz
dl.ocanoke.com
dogemanager.com
fodz.xyz
iddeas.xyz
idzone.site
iloveamritsar.in
kayisi.top
mysteriousbox.in
onelink.pw
salee.xyz
sparkly.to
teachersjobindia.com
techshine.xyz
thenewyear.xyz
trendbars.com
trendgang.info
url.topadnetworks.net
zend.phytech.ro
zend.theplusit.ro
44car.site
girlxxx.org
trickspremium.ga
api.grhls.stream
doxxan.club
noxx.to
infotekno.net
go.infotekno.net
uwatchfreetv.cf
dotpelangi.xyz
payco.xyz
realcam.pw
dongyoungsang.me
datpiff.fun
siteneverdown.xyz
funnygames.vn
data-raid-recovery-link.com
! Many subdomain, all tested were dead
template-download.top
pesnewupdate.click
freeapkgame.net
msphdscholarship.com
streaming2040.org
xyzbikini.com

! ###adsense was removed from EL so removed the whole rule
9453av18.net
ahsentv.com.tr
aikido-ueshiba.com
allwallpapers.info
anibatch.id
animesubindo.id
arabsex.be
avselected.com
bakifail.hu
behshaddavoudi.ir
beritadumay.com
beurettefacile.com
bitnet-co.ir
bleachmx-news.net
classtonic.com
club-quant.ru
coolvideo.info
crackplaza.com
dabarecrutement.com
debcstmdsgn.com
devassas.net
dogesurf.com
exhdmovies.in
fashandmag.com
flightfindair.com
footballatclick.com
funkysnippet.nl
geeknews.click
gestiondesti.com
getmyfile.ga
gfxfree.tk
gfxpirate.com
ginogy.com
gojio.net
hawkfeather.wp-x.jp
hostingtop100.com
humandroid.com.br
idol48fans.web.id
imoham3d.ml
indiaroadtrips.com
interfirstinsurance.com
iplliveupdate.in
jokerfansub.top
jokerfansub.xyz
justonemoreday.info
kalkographus.pl
keshavcommoditycalls.com
keygen-download.de
klikni-i-uzivaj.com
klu15.in
koungeek.com
leakedearly.gdn
leakedearly.review
legalescrow.co.uk
livingin2k.net
livrosconservadores.com.br
lobbystas.gr
lolnews2030.com.br
loukosporandroid.cc
luzoplanet.com
master-develop.com
mawqii.com
mp3-telechargement.net
nomascrisis.com
nudefoto.club
nulledradar.com
onlinemakejobs.com
photojav.com
pornoparaelas.com
purwaka.net
qstage.com.br
r15team.net
revdlz.com
roofvis.tv
sexonlineporn.com
shatimes.com
sledujufilmyserialy.cz
socialextensions.eu
tacobit.website
tamilguns.in
technicalscoop.com
tecnologosrd.com
theurbanist.info
thewebhow.com
trapress.net
turkifsalarizle.com
turkishyama.com
venpacks.com.ve
veryhtml5.com
vskoole.net
weeds-streaming.fr
wololo-project.es
xclub.sk
xdiggers.com
xpenology-fr.net
xturkifsa.org
xxxhdonline.com
zippysharemediafire.download
!
facildownloads.net
spillvacation.com
manastaynight09.blogspot.kr
kmspico.esy.es
fulltvsports.com
avxwcuaagfcdn.net
envatostuff.rocks
dzxxxg6ij9u99.cloudfront.net
ifirstrowpt.eu


! ---------- Now in EL (can be slightly different form)
||allartand.club^
||fandmo.com^
||leonodikeu9sj10.com^
||mehebborc.com^
||tubeadvertising.eu^$third-party
||upgulpinon.com^
||zuphaims.com^

! (dead)
||euktoywjpdxpdh.com^
||gzxdkaytxdsyxg.com^
||horheloopo.com^
||kuchebraska.com^
||nuhwyavc.com^
||prikolizdesa.com^
||reqpostanza.com^
||shmonekisot.com^
||tsastrmq.com^

||d261u4g5nqprix.cloudfront.net^
||d2fbkzyicji7c4.cloudfront.net^
||djv99sxoqpv11.cloudfront.net^
||dkyp75kj7ldlr.cloudfront.net^
||ds88pc0kw6cvc.cloudfront.net^

! covered by EL filter washingtonpost.com##[style*="padding-left: 10px; padding-right: 10px;"]
washingtonpost.com#?#div[style^="padding-left:"] > div[style="min-height: 90px;"] > wp-ad[class]:upward(1)


! ---------- Obsolete rules
speed-down.org#%#//scriptlet("prevent-bab")
```

### CODE REVIEW

**Comment by Yuki2718** [comment]
@ryanbr These domain in EL seem to be dead:
```
||euktoywjpdxpdh.com^
||gzxdkaytxdsyxg.com^
||horheloopo.com^
||kuchebraska.com^
||nuhwyavc.com^
||prikolizdesa.com^
||reqpostanza.com^
||shmonekisot.com^
||tsastrmq.com^
```

**Comment by zloyden** [comment]
I agree with the most of changes here, but `speed-down.org#%#//scriptlet("prevent-bab")` this is not true, you should restore it.
What did you change in `--- KillAdBlock ---`? I restored it because of conflicts.

**Comment by zloyden** [comment]
Also restored `Force adblock` section for the same reason. If you want make changes here,  you better tell me exactly what you want to change. It's better do directly (not here) to avoid these conficts.

**Comment by Yuki2718** [comment]
You're right, bab script is still there - IDK why I missed that. I want to remove these dead domain from filters:
```
animeheaven.eu
animeupdate6.pro
bitcomania.io
counsellingresult2016.in
dasffad2590.ramrem.space
forbes.uol.com.br
funnyquiz.blog
linksgenerator.ga
megapelishdonline.website
ngantukberat.me
repo.xarold.com
safebit.co.in
semimkv.net
techwidetoday.site
darkcomet.net
firmgoogle.com
apprendre-les-runes.com
arenavision2017.cf
dicaspravida.life
file4go.me
hibrido1337.blogspot.com
indextred.xyz
newsdust9.live
oldjogos.com
rl-hackers.net
files.love
2url.io
artiicle.xyz
bollyleague.com
craftz.xyz
dl.ocanoke.com
dogemanager.com
fodz.xyz
iddeas.xyz
idzone.site
iloveamritsar.in
kayisi.top
mysteriousbox.in
onelink.pw
salee.xyz
sparkly.to
teachersjobindia.com
techshine.xyz
thenewyear.xyz
trendbars.com
trendgang.info
url.topadnetworks.net
zend.phytech.ro
zend.theplusit.ro
44car.site
girlxxx.org
trickspremium.ga
api.grhls.stream
doxxan.club
noxx.to
infotekno.net
go.infotekno.net
uwatchfreetv.cf
dotpelangi.xyz
payco.xyz
realcam.pw
dongyoungsang.me
datpiff.fun
siteneverdown.xyz
funnygames.vn
data-raid-recovery-link.com
! Many subdomain, all tested were dead
template-download.top
pesnewupdate.click
freeapkgame.net
msphdscholarship.com
streaming2040.org
xyzbikini.com
```
Tell me if I can do something.

**Comment by zloyden** [comment]
@Yuki2718 https://github.com/AdguardTeam/AdguardFilters/pull/69473#issuecomment-740535646 let's make another PR for this list, there is a lot of changes.

File: EnglishFilter/sections/adservers.txt [hunk 1]
```diff
@@ -2528,7 +2497,6 @@ www.llllllllllll.net^$third-party
 ||u1trkqf.com^$redirect=nooptext
 ||aanqylta.com^$redirect=nooptext
 ||35.188.14.213^$redirect=nooptext
-||2778255fe56.com^$redirect=nooptext
```
[the remark this answers is not in the document]
Comment (Yuki2718) [reply EnglishFilter/sections/adservers.txt:2531]: Why restore? It's dead domain. E.g. `http://ww1.2778255fe56.com/`

[the remark this answers is not in the document]
Comment (zloyden) [reply EnglishFilter/sections/adservers.txt:2531]: Good catch )

**Comment by Alex-302** [comment]
Better write domains to remove in ant-adblock rules. This is impossible to compare if conflicts will be added.

**Comment by Yuki2718** [comment]
As you see, all domains I wanna remove are in the first post space separated for each file.

**Comment by Alex-302** [comment]
I don't understand why some rules was removed completely in antiadblock.txt

**Comment by Yuki2718** [comment]
```
animeheaven.eu
animeupdate6.pro
bitcomania.io
counsellingresult2016.in
dasffad2590.ramrem.space
forbes.uol.com.br
funnyquiz.blog
linksgenerator.ga
megapelishdonline.website
ngantukberat.me
repo.xarold.com
safebit.co.in
semimkv.net
techwidetoday.site
darkcomet.net
firmgoogle.com
apprendre-les-runes.com
arenavision2017.cf
dicaspravida.life
file4go.me
hibrido1337.blogspot.com
indextred.xyz
newsdust9.live
oldjogos.com
rl-hackers.net
files.love
2url.io
artiicle.xyz
bollyleague.com
craftz.xyz
dl.ocanoke.com
dogemanager.com
fodz.xyz
iddeas.xyz
idzone.site
iloveamritsar.in
kayisi.top
mysteriousbox.in
onelink.pw
salee.xyz
sparkly.to
teachersjobindia.com
techshine.xyz
thenewyear.xyz
trendbars.com
trendgang.info
url.topadnetworks.net
zend.phytech.ro
zend.theplusit.ro
44car.site
girlxxx.org
trickspremium.ga
api.grhls.stream
doxxan.club
noxx.to
infotekno.net
go.infotekno.net
uwatchfreetv.cf
dotpelangi.xyz
payco.xyz
realcam.pw
dongyoungsang.me
datpiff.fun
siteneverdown.xyz
funnygames.vn
data-raid-recovery-link.com
! Many subdomain, all tested were dead
template-download.top
pesnewupdate.click
freeapkgame.net
msphdscholarship.com
streaming2040.org
xyzbikini.com
```
and
```
facildownloads.net
spillvacation.com
manastaynight09.blogspot.kr
kmspico.esy.es
fulltvsports.com
avxwcuaagfcdn.net
envatostuff.rocks
dzxxxg6ij9u99.cloudfront.net
ifirstrowpt.eu
```
need to be removed individually. No need to remove
```
9453av18.net
ahsentv.com.tr
aikido-ueshiba.com
allwallpapers.info
anibatch.id
animesubindo.id
arabsex.be
avselected.com
bakifail.hu
behshaddavoudi.ir
beritadumay.com
beurettefacile.com
bitnet-co.ir
bleachmx-news.net
classtonic.com
club-quant.ru
coolvideo.info
crackplaza.com
dabarecrutement.com
debcstmdsgn.com
devassas.net
dogesurf.com
exhdmovies.in
fashandmag.com
flightfindair.com
footballatclick.com
funkysnippet.nl
geeknews.click
gestiondesti.com
getmyfile.ga
gfxfree.tk
gfxpirate.com
ginogy.com
gojio.net
hawkfeather.wp-x.jp
hostingtop100.com
humandroid.com.br
idol48fans.web.id
imoham3d.ml
indiaroadtrips.com
interfirstinsurance.com
iplliveupdate.in
jokerfansub.top
jokerfansub.xyz
justonemoreday.info
kalkographus.pl
keshavcommoditycalls.com
keygen-download.de
klikni-i-uzivaj.com
klu15.in
koungeek.com
leakedearly.gdn
leakedearly.review
legalescrow.co.uk
livingin2k.net
livrosconservadores.com.br
lobbystas.gr
lolnews2030.com.br
loukosporandroid.cc
luzoplanet.com
master-develop.com
mawqii.com
mp3-telechargement.net
nomascrisis.com
nudefoto.club
nulledradar.com
onlinemakejobs.com
photojav.com
pornoparaelas.com
purwaka.net
qstage.com.br
r15team.net
revdlz.com
roofvis.tv
sexonlineporn.com
shatimes.com
sledujufilmyserialy.cz
socialextensions.eu
tacobit.website
tamilguns.in
technicalscoop.com
tecnologosrd.com
theurbanist.info
thewebhow.com
trapress.net
turkifsalarizle.com
turkishyama.com
venpacks.com.ve
veryhtml5.com
vskoole.net
weeds-streaming.fr
wololo-project.es
xclub.sk
xdiggers.com
xpenology-fr.net
xturkifsa.org
xxxhdonline.com
zippysharemediafire.download
```
as it's better to remove the whole rule for `#@##adsense`

**Comment by Yuki2718** [comment]
> I don't understand why some rules was removed completely in antiadblock.txt

Example?

File: EnglishFilter/sections/adservers.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (Alex-302) [reply EnglishFilter/sections/adservers.txt:2531]: image
Looks like clickunder/popup related

**Comment by Alex-302** [comment]
> Example?

`anoptions`  rules. Now they added again.

**Comment by Alex-302** [comment]
I mean `! Adblock Notify plugin https://themeisle.com/plugins/adblock-notify/` in the message above)

**Comment by Yuki2718** [comment]
`###adsense` was removed from EL so the whitelist is no more needed.

**Comment by Alex-302** [comment]
Ah, now got it) Thanks.

**Review by Alex-302 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Comment by Yuki2718** [comment]
Thx!

**Comment by Alex-302** [comment]
It seems we can remove all `#@##adsense` from filters

**Comment by Yuki2718** [comment]
Some domains intended to be removed were also restored to avoid conflicts e.g. `firmgoogle.com`. Happy if you can remove them, but within a degree not interrupting your job.

**Comment by Alex-302** [comment]
Ok.

**Comment by zloyden** [comment]
I will check it today.

**Comment by Yuki2718** [comment]
You don't need to check
```
facildownloads.net
spillvacation.com
manastaynight09.blogspot.kr
kmspico.esy.es
fulltvsports.com
avxwcuaagfcdn.net
envatostuff.rocks
dzxxxg6ij9u99.cloudfront.net
ifirstrowpt.eu
```
Only the first paragraph of https://github.com/AdguardTeam/AdguardFilters/pull/69473#issuecomment-740541255 is worth checked.

**Comment by zloyden** [comment]
> Only [#69473 (comment)](https://github.com/AdguardTeam/AdguardFilters/pull/69473#issuecomment-740535646) is worth checked.

I meant this list. so I removed those domains (which are existed in our filters). Thanks)

**Pull request merged**